Document Android multi process Apps for Android SDK (#2898)
getsentry/sentry-docs: src/platforms/android/usage/advanced-usage.mdx

</application>
```

#### Multi-Process Apps

> Due to using `ContentProvider`, Sentry Android SDK will not automatically initialize in non-main processes.
> 
> If you'd like to automatically `auto-init` the Sentry Android SDK in other processes, add this to your manifest:

Reviewer: ```suggestion
The Sentry Android SDK will not automatically initialize in non-main processes because of `ContentProvider`. If you'd like to automatically `auto-init` the Sentry Android SDK in other processes, add the following to your manifest: 
```

```xml {filename:AndroidManifest.xml}
<provider
  android:name="io.sentry.android.core.SentryInitProvider"
  android:authorities="${applicationId}.SentryInitProvider"
  android:exported="false"
  android:process="YOUR_PROCESS_HERE"
  tools:node="merge"/>
```

> Alternatively, you can call `SentryAndroid.init(...)` yourself.

Reviewer: ```suggestion
Alternatively, you can call `SentryAndroid.init(...)`.
```
